src/sentry/options/defaults.py

```diff
@@ -26,7 +27,7 @@
 # Absolute URL to the sentry root directory. Should not include a trailing slash.
 register('system.url-prefix', ttl=60, grace=3600, flags=FLAG_REQUIRED | FLAG_PRIORITIZE_DISK)
 register('system.root-api-key', flags=FLAG_PRIORITIZE_DISK)
-register('system.logging-format', default='human', flags=FLAG_PRIORITIZE_DISK)
+register('system.logging-format', default=LoggingFormat.HUMAN, flags=FLAG_PRIORITIZE_DISK)
```

Comment:
We'll later want to add an num type to the options store that understands this.

Right now, it'd be possible to do:

```yaml
system.logging-format: lol
```

and things would carry on. Outside this scope though, just leaving it as a note.